packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -1,5 +1,13 @@
-import { MouseEventHandler, useEffect, useMemo } from 'react';
+import {
+  MouseEventHandler,
+  useEffect,
+  useMemo,
+  useState,
+  useRef,
+  SyntheticEvent,
+} from 'react';
 import classnames from 'classnames';
+import { debounce } from 'lodash';
```

Comment:
```suggestion
import debounce from 'lodash/debounce';
```

Reply:
Thank you @Rhotimee for the suggestions! Done here - 63e83369db138986fa5abe2e7b361dbc5bf3018d.